src/tools/vtpipeterm/res.rc

```diff
@@ -1,16 +0,0 @@
-// Copyright (c) Microsoft Corporation.

```

Comment:
we could keep this, it is OK

Reply:
Oh, I forgot about this comment.

I initially thought this was a leftover file for translatable strings, but now I realize that this is for the .exe information. Hmmm, I think it's fine to remove in that case, since despite this tool being part of the OS repo, I'm not sure it's used by anyone there anyway.